docs-site/docs/development/development_processes/pull_request_workflow.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 2
+id: pull_request_workflow
+title: Pull requests workflow
+---
+
+We welcome pull requests from everyone, with fixes and improvements to IRIDA Next code, tests, and documentation.
+
+## Merge request guidelines for contributors
+
+### Best practices
+
+* If the change is non-trivial, we encourage you start a discussion with a team member. You can do this by tagging them in a PR before submitting the code for review. Taking to team embers can be helpful when making design decisions. Communicating the intent behind your changes can also help expedite pull request reviews.
+* When having your code reviewed and when reviewing pull requests, please keep the [code review guidelines](./code_review) in mind.
```

Comment:
This link appears to be broken for me.

Reply:
Can you elaborate, are you saying when you run the docs site that the link doesn't work? The link works normally from me when running the docs site with `pnpm run start` inside of the `docs-site` folder.